src/cpp/src/icontinuous_batching.cpp

```diff
@@ -179,6 +217,7 @@ ContinuousBatchingPipeline::IContinuousBatchingPipeline::add_request(uint64_t re
     ov::Tensor inputs;
     {
         const std::lock_guard<std::mutex> lock(m_inputs_embedder_mutex);
+        m_inputs_embedder->set_apply_chat_template_status(sampling_params.apply_chat_template);
```

Comment:
`add_request` assumes that chat template is externally applied

Reply:
@mzegla for chat scenarios you need to apply chat template externally and pass `apply_chat_template=False` in generation config.